osgp/protocol-adapter-iec60870/pom.xml

```diff
@@ -18,7 +18,7 @@
   <packaging>pom</packaging>
   <name>protocol-adapter-iec60870</name>
   <properties>
-    <jxr.version>2.5</jxr.version>
+    <jxr.version>3.0.0</jxr.version>
```

Comment:
When generating a maven site, this `property jxr.version` is required, along with `maven.project.info.reports.plugin.version` and `maven.site.plugin`. Since the super pom project can't use the dependency / plugin versions defined in the parent pom, these 3 properties are also present in all super pom files.